packages/commerce-sdk-react/src/auth/storage.ts

```diff
@@ -9,25 +9,23 @@ import Cookies from 'js-cookie'
 export type StorageType = 'cookie' | 'local' | 'memory'
 
 export interface BaseStorageOptions {
-    keyPrefix?: string
-    keyPrefixSeparator?: string
```

Comment:
remove the option as change the separator will break the compatibility with plugin_slas, we don't want to give user the option to do this.